doc/flame/examples/pubspec.yaml

```diff
@@ -4,8 +4,8 @@ version: 1.0.0
 publish_to: none
 
 environment:
-  sdk: ^2.16.0
-  flutter: ^2.10.0
+  sdk: ">=2.17.0 <3.0.0"
+  flutter: ">=3.0.0"
```

Comment:
Why write `">=3.0.0"` instead of `^3.0.0` ?

Reply:
Just because it was like that everywhere else, so it makes it easier to search and replace, but I could change all other to `^3.0.0` instead.